src/service/Utils.ts

```diff
@@ -0,0 +1,27 @@
+import { RuntimeService } from './RuntimeService';
+
+/**
+ * Random utility methods that don't fit other place.
+ */
+export class Utils {
+    public static secureString(text: string): string {
+        const regex = new RegExp('[0-9a-fA-F]{64}', 'g');
+        return text.replace(regex, 'HIDDEN_KEY');
+    }
+
+    public static validateIsDefined(value: unknown, message: string): void {
+        if (value === undefined || value === null) {
+            throw new Error(message);
+        }
+    }
+
+    public static validateIsTrue(value: boolean, message: string): void {
+        if (!value) {
+            throw new Error(message);
+        }
+    }
+    public static logSameLineMessage(message: string): void {
+        process.stdout.write(RuntimeService.isWindows() ? '\x1b[0G' : '\r');
```

Comment:
dependency on RuntimeService seems unnatural. probably better to move isWindows into this class.

Reply:
Created OSUtils